Week3

```diff
@@ -0,0 +1,64 @@
+// Homework 2 
+// Part 1
+Create a Car class.
+Create the following attributes:
+Country of car
+Model
+Maker
+
+Create methods:
+Set country – should set a country for car object
+Set model – should set a model for car object
+Set maker – – should set a maker for car objectGet country – Should return country of car
+Get model – Should return model of car
+Get maker – – Should return maker of car
+
+public class Main {
+
+    public static void main(String[] args) {
+        class Car {
+
+            String country;
+            String model;
+            String maker;
+
+            public String getCountry(){
+                return country;
+            }
+
+            public void setCountry(String newCountry){
+                country = newCountry;
+            }
+
+            public String getModel(){
+                return model;
+            }
+
+            public void setModel(String newModel){
+                model = newModel;
+            }
+
+            public String getMaker(){
+                return maker;
+            }
+
+            public void setMaker(String newMaker){
+                maker = newMaker;
+            }
+        }
+
+        class Solution {
```

Comment:
It is very bad way to implement new classes into main method. Please remove it.

Reply:
class Solution ubrat'?

Reply:
yes, off couse!!! Solution class ubrat'!
this code:
```
Car audi = new Car();
                audi.setCountry("United States");
                audi.setModel("2020");
                audi.setMaker("Audi");

                System.out.println(audi.getCountry());//United States
                System.out.println(audi.getModel());//2020
                System.out.println(audi.getMaker()); //Audi

```
you need to extract from `public void main(String[] args)` method in Solution class and placing to the `public static void main(String[] args)` instead classes Car and Solution. 

```
And class Car:
class Car {
            String country;
            String model;
            String maker;

            public String getCountry(){
                return country;
            }

            public void setCountry(String newCountry){
                country = newCountry;
            }

            public String getModel(){
                return model;
            }

            public void setModel(String newModel){
                model = newModel;
            }

            public String getMaker(){
                return maker;
            }

            public void setMaker(String newMaker){
                maker = newMaker;
            }
        }
```
you need to write not into `public static void main(String[] args)` method but in the separate file or in this Main class but not in 'main' method.